mne/decoding/time_gen.py

```diff
@@ -1375,3 +1466,17 @@ def _clean_times(self):
             self.y_pred_ = [y_pred[0] for y_pred in self.y_pred_]
         if hasattr(self, 'scores_'):
             self.scores_ = [score[0] for score in self.scores_]
+
+
+def _chunk_X(X, slices, gat, n_orig_epochs, test_epochs):
+    """Smart chunking to avoid memory overload"""
```

Comment:
can you add a line to explain why it is smart / the idea behind how you avoid memory overload